src/routes/alerts/entities/schemas/alerts.schema.ts

```diff
@@ -77,8 +77,8 @@ export const alertSchema: JSONSchemaType<Alert> = {
     id: { type: 'string' },
     event_type: {
       type: 'string',
-      default: AlertEventType.ALERT,
-      enum: Object.values(AlertEventType),
+      default: EventType.ALERT,
```

Comment:
OOC: why do we assume a default for the `EventType`?

Reply:
We shouldn't be - good catch! There is also a `"TEST"` `type`. I've removed the `default` in 07d22ecc4f5541ec9dfbde1722174cf7ce469364.

Reply:
We shouldn't be - good catch! There is also a `"TEST"` `type`. I've removed the `default` in 07d22ecc4f5541ec9dfbde1722174cf7ce469364.